src/parser/postgresparser.cpp

```diff
@@ -1071,7 +1071,11 @@ parser::SQLStatement *PostgresParser::CreateTriggerTransform(
 parser::SQLStatement *PostgresParser::CreateDbTransform(CreatedbStmt *root) {
   parser::CreateStatement *result =
       new parser::CreateStatement(CreateStatement::kDatabase);
-  result->database_name = root->dbname;
+  result->table_info_.reset(new parser::TableInfo());
+  result->table_info_->database_name = root->dbname;
+
+  // TODO(Tianyi) More options need to be converted
```

Comment:
Such as?

Reply:
There is a list of options in the CreateDbNode and they are discarded in the node transform.
https://github.com/cmu-db/peloton/blob/master/src/include/parser/parsenodes.h#L625

I think these options support more complex create database queries mentioned in https://www.postgresql.org/docs/10/static/sql-createdatabase.html

I can add the above reference to the comments here in case others can take a chance and implement it.